.tekton/push.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     - name: revision
       value: "{{revision}}"
     - name: output-image
-      value: "quay.io/redhat-appstudio/sprayproxy:{{revision}}"
+      value: "quay.io/konflux-ci/sprayproxy:{{revision}}"
```

Comment:
Is there konflux-ci org in Quay? It either does not exist or it's me not having permissions to confirm.

Reply:
Even I don't see the `konflux-ci` in quay.

Reply:
so is the plan to keep the image under redhat-appstudio ?